frontend/src/app/supportAdmin/PendingOrganizations/PendingOrganizations.tsx

```diff
@@ -255,9 +256,9 @@ const PendingOrganizations = ({
               />
             ) : null}
             <div className="usa-card__header">
-              <h2 data-cy="pending-orgs-title">
+              <h1 data-cy="pending-orgs-title">
                 Edit or verify organization identity
```

Comment:
Style changed here as well... just flagging in case we want to change it to be more similar to before :) 

Before:
<img width="1451" alt="Screen Shot 2022-11-21 at 11 03 50" src="https://user-images.githubusercontent.com/20211771/203116161-8805343c-f514-4ede-b593-1db1be5aa15c.png">

After:
<img width="1888" alt="Screen Shot 2022-11-21 at 10 58 26" src="https://user-images.githubusercontent.com/20211771/203116269-576a1cd9-dfdb-4e79-a737-2919cfd4363c.png">